app.js

```diff
@@ -155,31 +163,33 @@ function LeftButton(route, navigator, index, navState) {
       if (index === 1) {
         backTitle = 'Home'
       }
-      return (
-        <TouchableOpacity
-          style={styles.backButton}
-          onPress={() => navigator.pop()}
-        >
-          <Icon style={styles.backButtonIcon} name='ios-arrow-back' />
-          <Text style={styles.backButtonText}>{backTitle}</Text>
-        </TouchableOpacity>
-      )
+      if (Platform.OS === 'android') {
+        return (
+          <TouchableOpacity
+            style={styles.backButton}
+            onPress={() => navigator.pop()}
+          >
+              <Icon style={styles.backButtonIcon} name='md-arrow-back' />
```

Comment:
Can you move it down, then? (did you already fix that?)

It's really odd that it doesn't show up in the sim. I'll dig into that a bit tonight.